### ISSUE TITLE
QA - [Contributing / Ad-ons]: The green is too dark on check icons

### ISSUE THREAD

**Issue by mjabbink** [carbon-design-system/carbon-website#1048]
@jeanservaas @shixiedesign 

<img width="487" alt="screen shot 2019-02-09 at 8 49 02 am" src="https://user-images.githubusercontent.com/32881239/52522207-ae4b9580-2c47-11e9-9ede-d51dd399465a.png">

**Comment by shixiedesign** [comment]
@mjabbink we made a mistake in token assignment. Should be `Green 50` (3:1 contrast ratio needed so cannot go any lighter). You should see the update soon.
